pytensor/link/numba/dispatch/scan.py

```diff
@@ -112,25 +112,46 @@ def numba_funcify_Scan(op, node, **kwargs):
     # Inner-inputs are ordered as follows:
     # sequences + mit-mot-inputs + mit-sot-inputs + sit-sot-inputs +
     # shared-inputs + non-sequences.
+    temp_scalar_storage_alloc_stmts: List[str] = []
+    inner_in_exprs_scalar: List[str] = []
     inner_in_exprs: List[str] = []
 
     def add_inner_in_expr(
-        outer_in_name: str, tap_offset: Optional[int], storage_size_var: Optional[str]
+        outer_in_name: str,
+        tap_offset: Optional[int],
```

Comment:
The `else` branch accepts `None` no?

Reply:
It errors before it gets there because `int < None` is invalid

Reply:
The `tap_offset` None prevents the call to `idx_to_str` in the else branch. In the `is_vector` branch (which I now rename dto `vector_slice_opt` it would error out, but `None` taps are not valid in that branch.